src/components/AssetInput/AssetInput.stories.tsx

```diff
@@ -117,40 +68,169 @@ PlaceholderLibraryOnly.argTypes = {
 export const Image = Template.bind({});
 
 Image.args = {
-    asset: {
-        source: "upload",
-        name: "foo",
-        size: 2000,
-        type: "image",
-        extension: "JPG",
-        src: "https://picsum.photos/200/300",
-    },
+    assets: [
+        {
+            source: "upload",
+            name: "foo",
+            size: 2000,
+            type: "image",
+            extension: "JPG",
+            src: "https://picsum.photos/200/300",
+        },
+    ],
     actions,
 };
 
 export const Audio = Template.bind({});
 
 Audio.args = {
-    asset: {
-        source: "library",
-        sourceName: "Foobar",
-        name: "foo",
-        extension: "MP3",
-        size: 2000,
-        type: "audio",
-    },
+    assets: [
+        {
+            source: "library",
+            sourceName: "Foobar",
+            name: "foo",
+            extension: "MP3",
+            size: 2000,
+            type: "audio",
+        },
+    ],
     actions,
 };
 
 export const Icon = Template.bind({});
 
 Icon.args = {
-    asset: {
-        source: "library",
-        sourceName: "Foobar",
-        name: "foo",
-        type: "icon",
-        icon: <IconIcons />,
-    },
+    assets: [
+        {
+            source: "library",
+            sourceName: "Foobar",
+            name: "foo",
+            type: "icon",
+            icon: <IconIcons />,
+        },
+    ],
     actions,
 };
+
+const TemplateMultiInput: Story<AssetProps> = (args) => {
```

Comment:
I think you could reuse the same Template as in the stories above

Reply:
I once tried – didn't work. Now tried again – works smoothly. Don't know what I tried back than 😂

Reply:
Nice catch. I completely forgot it